lib/plottr_components/src/components/project/BookDialog.js

```diff
@@ -64,11 +99,12 @@ const BookDialogConnector = (connector) => {
             </Col>
             <Col sm={8}>
```

Comment:
(This is a long-standing problem): at some point, we need to change this into a controlled component.
Right now, it uses refs to grab the value when we save.
For these controls to work well with Pro, they need to update the stare with every key-stroke.